### PR TITLE
Attempted fix for initial load issue.

### LINKED ISSUE #48: Issue with initial load from iCloud
On an initial install of an app with Zephyr the iCloud data is not immediately available. Once it is available is should be synced with the current `UserDefaults`. This does not happen on an initial load however, as `keysDidChangeOnCloud` will return early if `localStoredDate` is nil (which it is as this is the initial load of the application). 

The issue seems to be in `keysDidChangeOnCloud:452`, have made a PR: #47.

### PR DESCRIPTION
**Pull request by aedificator-nl**
All parts of the guard statement have been split into let statements and two separate if statements.

If `localStoredData == nil` we still want to synchronize.

See issue #48.

### CODE REVIEW

File: Sources/Zephyr.swift [hunk 1]
```diff
@@ -447,12 +447,17 @@ extension Zephyr {
...
+            let userInfo = (notification as NSNotification).userInfo
+            let cloudKeys = userInfo?[NSUbiquitousKeyValueStoreChangedKeysKey] as? [String]
+            let localStoredDate = zephyrLocalStoreDictionary[ZephyrSyncKey] as? Date
+            let remoteStoredDate = zephyrRemoteStoreDictionary[ZephyrSyncKey] as? Date
+            
+            if (userInfo == nil || remoteStoredDate == nil || cloudKeys == nil) {
+                return
+            }
+            
+            if (localStoredDate != nil && remoteStoredDate != nil && !(remoteStoredDate!.timeIntervalSince1970 > localStoredDate!.timeIntervalSince1970)) {
```
Comment (ArtSabintsev) [inline Sources/Zephyr.swift:459]: these are force-unwrapped. Mind addressing that? Thank you!

Comment (aedificator-nl) [reply Sources/Zephyr.swift:459]: Absolutely. Have added extra check and avoid force-unwrapping.

Comment (ArtSabintsev) [reply Sources/Zephyr.swift:459]: `localStoredDate!` is still force unwrapped :)

Comment (aedificator-nl) [reply Sources/Zephyr.swift:459]: My bad, damn typos... Anyway, now it should actually be fixed ;)

Comment (ArtSabintsev) [reply Sources/Zephyr.swift:459]: Thanks - will check it out this weekend

**Comment by ArtSabintsev** [comment]
This does not compile in Xcode 11.5. A couple errors around optionals.

**Comment by aedificator-nl** [comment]
The force-unwrapping of the optionals is necessary to avoid Xcode throwing a fit. Given that a nil check has already been done the code that force-unwraps is not reachable with a nil value, thus never throwing an error. 

To me this seems the best way to use the optional values here. Do you have a suggestion for a better way to approach this? Thanks in advance!

File: Sources/Zephyr.swift (hunk 1 above)
Comment (aedificator-nl) [reply Sources/Zephyr.swift:459]: We have force-unwrapping here again, see comment below.

**Comment by ArtSabintsev** [comment]
I'll fix the force-unwrap issue once I compile and test your code in a branch of mine. Thanks!

**Comment by aedificator-nl** [comment]
Thanks! And thanks again for the whole project, awesome piece of code :)

**Comment by ArtSabintsev** [comment]
Thank you! I appreciate that!

**Comment by ArtSabintsev** [comment]
Alright, tested it in a person project - worked well. Will merge your stuff then overwrite with a cleaner implementation that has no force-uwnraps. Thanks again!

**Comment by ArtSabintsev** [comment]
Here's the release notes: https://github.com/ArtSabintsev/Zephyr/releases/tag/3.6.0

It should be available across all dependency managers now.

**Comment by aedificator-nl** [comment]
No problem, glad to be able to help. Thank you for the kind mention in the release notes!

**Pull request merged**